packages/gateway-trpc/src/modules/event/event-router.ts

```diff
@@ -58,6 +58,18 @@ export const eventRouter = t.router({
       committees: results[i],
     }))
   }),
+  recommended: publicProcedure.query(async ({ ctx }) => {
+    const events = await ctx.eventService.getEvents(4)
+    const committees = events.map(async (e) => ctx.eventCommitteeService.getEventCommitteesForEvent(e.id))
```

Comment:
N+1 Query

Reply:
True that, kokte fra all routen, men burde fikses